apps/condo/domains/common/components/Comments/Comment.tsx

```diff
@@ -95,17 +95,22 @@ export const Comment: React.FC<ICommentProps> = ({ comment, updateAction, delete
 
     const [mode, setMode] = useState<CommentMode>('display')
     const [content, setContent] = useState(comment.content)
+    const [editCanceled, setEditCanceled] = useState<boolean>(false)
```

Comment:
If you provide `false` as default values, hook will automatically generate boolean type